EpiserverRedirects/ClientResources/UrlRedirectsMenu/UrlRedirectsMenu.js

```diff
@@ -1,4 +1,4 @@
-define("urlRedirectsMenu/UrlRedirectsMenu", [
```

Comment:
Would it be a lot of work to have all static files that module depends on packed in zip archive (like for all other Episerver modules)? It would not generate lots of changes in different files when updating this package, but only single contents zip file

Reply:
Oki, so right now all static files are in the zip archive